src/dsymbol/symbol.d

```diff
@@ -429,15 +426,9 @@ public:
 
 struct UpdatePair
 {
-	int opCmp(ref const UpdatePair other) const pure nothrow @nogc @safe
+	ptrdiff_t opCmp(ref const UpdatePair other) const pure nothrow @nogc @safe
 	{
-		immutable size_t otherOld = cast(size_t) other.oldSymbol;
```

Comment:
why "other " value was used for "greater" ? does it matter here ?

Reply:
I don't think so.